health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreenParent/AppFieldComposer.js

```
@@ -0,0 +1,600 @@
+                  gap: "0.5rem",
+                  alignItems: "center",
+                  marginTop: "1rem",
+                }}
+              >
+                <DustbinIcon />
+              </div>
+            )}
+          </LabelFieldPair>
+        </>
+      );
+      break;
```

Biome (1.9.4)

[error] This code is unreachable



(lint/correctness/noUnreachable)